src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -87,7 +88,8 @@ def _make_causal_mask(input_ids_shape: tf.TensorShape, past_key_values_length: i
     """
     Make causal mask used for bi-directional self-attention.
     """
-    bsz, tgt_len = input_ids_shape
+    bsz = input_ids_shape[0]
+    tgt_len = input_ids_shape[1]
```

Comment:
dynamic shapes (i.e. the output of tf.shape, as opposed to tensor.shape) are not iterable